pkg/runner/cluster_k8s.go

```diff
@@ -976,9 +968,15 @@ func (c *ClusterK8sRunner) checkClusterResources(ow *rpc.OutputWriter, groups []
 
 	// all worker nodes are the same, so just take allocatable CPU from the first
 	item := res.Items[0].Status.Allocatable["cpu"]
-	nodeCPUs, _ := item.AsInt64()
+	nodeCPUs, success := item.AsInt64()
+
+	// in case parsing fails, the milli value will contain the proper format of cpu resources
+	if !success {
+		nodeCPUs = item.MilliValue()
+	}
```

Comment:
How does `AsInt64` fail? Does it depend on the architecture of the system you call it on?

Reply:
> How does AsInt64 fail?

I don't know - the only returned value is the `success` flag. And as I've said, I don't know how the parsing is handled (this is go's own module) to debug this. 

>Does it depend on the architecture of the system you call it on?

It would appear that it worked on the previous EC2 based cluster, but not on EKS, so the answer would appear to be yes.

Reply:
Don't we use 64 bit architecture for EKS machines?

Reply:
Yeah, we're using 64-bit architecture. [These](https://docs.aws.amazon.com/systems-manager/latest/userguide/prereqs-operating-systems.html) are the specs for some of the more popular distros on AWS.

Reply:
Yes.

Reply:
I will be pushing changes to this code block, and will resolve this conversation (as it is long enough already) in favor of starting a new one.

Reply:
https://github.com/testground/testground/pull/1350#discussion_r988095586